src/datasources/cache/entities/cache-dir.entity.ts

```diff
@@ -0,0 +1,9 @@
+export class CacheDir {
```

Comment:
I chose the name `CacheDir` but I'm open to suggestions. I also considered `CacheKey` but this kind of hides the fact the objects hold both `key` and `field`, also I thought about `CacheAddress` but the `address` concept is more heavily used in blockchain for other stuff.

Reply:
Another option, instead of having an ad-hoc interface, could be just returning a tuple `[key, field]` but I find it a bit more convoluted, less explicit, and harder to manage.

Reply:
Let's go with `CacheDir` for now. Another alternative could be `CachePath` (I see it as the "cache" path to the object that we want to retrieve).